roles/openshift_master/defaults/main.yml

```diff
@@ -6,7 +6,7 @@ os_firewall_allow:
 - service: etcd embedded
   port: 4001/tcp
 - service: api server https
-  port: 8443/tcp
+  port: 443/tcp
```

Comment:
We'll probably want to add 8443/tcp to the list of os_firewall_deny ports along with this change.